youtube_dl/extractor/servus.py

```diff
@@ -18,30 +18,35 @@ class ServusIE(InfoExtractor):
                         (?:www\.)?
                         (?:
                             servus\.com/(?:(?:at|de)/p/[^/]+|tv/videos)|
-                            (?:servustv|pm-wissen)\.com/videos
+                            (?:servustv|pm-wissen)\.com/(?:videos|[^/]+/v)
                         )
                         /(?P<id>[aA]{2}-\w+|\d+-\d+)
```

Comment:
When adding a additional `|\w+` for the id then also id's without a separator will work like the mentioned https://www.servustv.com/volkskultur/v/aawg5x4ywdc94xyjv9ly/

The full line should then be: `/(?P<id>[aA]{2}-\w+|\d+-\d+|\w+)`

Reply:
I was away and couldn't notice your review.

Reply:
Tested and works!